src/Dfc.ProviderPortal.TribalExporter/Services/VenueCollectionService.cs

```diff
@@ -79,5 +80,31 @@ public async Task<bool> HasBeenAnUpdatedSinceAsync(int ukprn, DateTime date)
 
             return documents.Count() > 0;
         }
+
+        public async Task<Venue> GetDocumentID(int venueId)
+        {
+            var documents = new List<Document>();
+
+            var uri = UriFactory.CreateDocumentCollectionUri(_cosmosDbSettings.DatabaseId, _cosmosDbCollectionSettings.VenuesCollectionId);
+            var sql = $"SELECT* FROM c WHERE c.VENUE_ID = { venueId}";
+            var options = new FeedOptions { EnableCrossPartitionQuery = true, MaxItemCount = -1 };
+
+            try
+            {
+
+                using (var client = _cosmosDbHelper.GetClient())
+                {
+                    var query = client.CreateDocumentQuery<Venue>(uri, sql, options).ToList();
+                    return query.FirstOrDefault();
+                };
+            }
+            catch (Exception e)
```

Comment:
Do we want to swallow the exception here?

Reply:
Good call, Removed now.